tools/generator/templates/stories/template.js.hbs

```diff
@@ -1,22 +1,29 @@
 import { html } from 'lit-html';
 import { classMap } from 'lit-html/directives/class-map.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
 
-// Uncomment if you plan to include an icon
-// import { Template as IconTemplate } from '../../icon/stories/template.js';
+import "../index.css";
 
 // More on component templates: https://storybook.js.org/docs/web-components/writing-stories/introduction#using-args
-export const Template = ({ rootClass, size }) => {
-  const classList = {
-    [rootClass]: true,
-    [`${className}--size${size.toUpperCase()}`]: true,
-  };
+export const Template = ({
+  rootClass = "spectrum-{{ pascalCase name }}",
+  size,
+  id,
+  customClasses = [],
+  ...globals
+}) => {
+  return html`
+    <div class=${classMap({
+      [rootClass]: true,
+      [`${className}--size${size.toUpperCase()}`]: true,
```

Comment:
```suggestion
      [`${rootClass}--size${size.toUpperCase()}`]: true,
```

I think we want `rootClass` here. As it is, `className` is returning `undefined`

Reply:
Oh yes! Great catch.